app/js/postEditFunction.js

```diff
@@ -0,0 +1,40 @@
+/*
+    Creates an object of the user from the data in the input fields
+    Should be called within the submit event listener
+
+    @param a string in the '.class' format, the class on each of the input fields of the edit user form
+
+    @return Returns an object that can be used as the parameter in postUserData() func.
+ */
+
+function createUserObject(inputClass) {
```

Comment:
VERY VERY semantically similar to createUsersObject in createUsersObject.js 
The difference in functionality is massive. Consider renaming